src/XliffTasks.Tests/ResxDocumentTests.cs

```diff
@@ -45,7 +45,40 @@ public void BasicLoadAndTranslate()
             document.Translate(translations);
             document.Save(writer);
 
-            Assert.Equal(expectedTranslation, writer.ToString());
+            AssertHelper.AssertWithoutLineEndingDifference(expectedTranslation, writer.ToString());
         }
+
+        [Fact]
+        public void RewriteFileReferenceToAbsoluteInDestinyFolder()
+        {
+            string source =
+@"<root>
+  <data name=""400"" type=""System.Resources.ResXFileRef, System.Windows.Forms"">
+    <value>Resources\Package.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
+  </data>
+</root>";
+
+            var translations = new Dictionary<string, string>
+            {
+                ["Hello"] = "Bonjour!",
+            };
+
+            string expectedTranslation =
+@"<root>
+  <data name=""400"" type=""System.Resources.ResXFileRef, System.Windows.Forms"">
+    <value>E:\sourceFolder\Resources\Package.ico;System.Drawing.Icon, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
+  </data>
+</root>";
+
+            var document = new ResxDocument();
+            var writer = new StringWriter();
+            document.Load(new StringReader(source));
+            document.RewriteRelativePathsToAbsolute(
+                @"E:\sourceFolder\Resources.resx");
+            document.Save(writer);
+
+            AssertHelper.AssertWithoutLineEndingDifference(expectedTranslation, writer.ToString());
```

Comment:
Can we use AssertEx.EqualIgnoringLineEndings like I suggested on the original PR?

Reply:
Sorry, I missed this change. I will add it

Reply:
fixed